lib/shopify_cli/tunnel.rb

```diff
@@ -188,14 +198,11 @@ def seconds_to_hm(seconds)
     def start_ngrok(ctx, port)
       process = ShopifyCLI::ProcessSupervision.start(:ngrok, ngrok_command(port))
       log = fetch_url(ctx, process.log_path)
-      seconds_remaining = (process.time.to_i + log.timeout) - Time.now.to_i
-      [log.url, log.account, seconds_remaining]
+      [log.url, log.account]
     end
 
     def restart_ngrok(ctx, port)
-      unless ShopifyCLI::ProcessSupervision.stop(:ngrok)
```

Comment:
What about keeping the `true/false` check. If I'm not wrong, I think the `stop` method returns `false` if the stopping failed. What I'd change though is the way we abort. Instead of aborting the execution directly from there, we should raise a `ShopifyCLI::Abort` error and let the error handler deal with the error:

```ruby
unless ShopifyCLI::ProcessSupervision.stop(:ngrok)
  raise ShopifyCLI::Abort, ctx.message("core.tunnel.error.stop")
end
```